sjb/config/test_cases/test_branch_origin_service_catalog.yml

```diff
@@ -0,0 +1,103 @@
+    - type: "script"
+      title: "update bindata"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        hack/update-generated-bindata.sh
+    - type: "script"
+      title: "build Origin OC binary with latest RBAC"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        OS_BUILD_ENV_PULL_IMAGE=true OS_BUILD_ENV_PRESERVE=_output/local/bin/linux/amd64/oc hack/env make build WHAT=cmd/oc
+    - type: "script"
+      title: "build service-catalog user-broker image"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        REGISTRY=openshift/ NO_DOCKER=1 make -C cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog user-broker-image
+    - type: "script"
+      title: "build Service Catalog binary"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        unset GOPATH; cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog/hack/build-go.sh
+    - type: "script"
+      title: "run Service Catalog unit tests"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        NO_DOCKER=1 make -C cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog test-unit
+    - type: "script"
+      title: "run Service Catalog integration tests"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        sudo yum -y install etcd
+        # this strips the -i arg to "go test". can be dropped once PR is merged:
+        # https://github.com/kubernetes-incubator/service-catalog/pull/1566 (and vendored in rebase)
+        sed -i 's/\(go test.*\)\( -i \)\(.*\)/\1 \3/' cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog/test/integration.sh
+        NO_DOCKER=1 make -C cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog scBuildImageTarget=.scBuildImage test-integration
+    - type: "script"
+      title: "build Service Catalog image"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        hack/build-local-images.py service-catalog
+    - type: "script"
+      title: "build Service Catalog E2E test binary"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        NO_DOCKER=1 make -C cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog  bin/e2e.test
+    - type: "script"
+      title: "start cluster with Service Catalog"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        ./_output/local/bin/linux/amd64/oc cluster up --version=latest --service-catalog
+    - type: "script"
+      title: "Run Service Catalog E2E"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        sudo chmod a+rw /var/lib/origin/openshift.local.config/master/admin.kubeconfig
+        export KUBECONFIG=/var/lib/origin/openshift.local.config/master/admin.kubeconfig
+        ./_output/local/bin/linux/amd64/oc login -u system:admin
```

Comment:
This `oc` should be in the `$PATH` if you care about making this a little shorter

Reply:
It actually didn't end up working without the full path, so I left it.